components/sensor/xiaomi_ble.rst

```diff
@@ -187,6 +187,11 @@ There are the following possibilities to operate this sensor:
    - "Mi Like" advertisement (dummy bindkey required)
    - "Custom" advertisement (no bindkey required)
    - "pvvx" custom advertisement (no bindkey required, only PVVX firmware)
+   - "reed-switch" is a binary_sensor that exposes the state of the pin labeled “P8”.
+         You have to have to be using the custom advertisement and solder on wires. See the section “
+         Reed Switch on GPIO PA6 (label on the "P8" pin)” on the pvvx github readme.
+         The state is only updated whenever the BLE is advertised.
+         Do not use it if timming is critical, for example, door or window sensors.
```

Comment:
please remove these margins, i.e. "You have..." must be right under the " above.

if you want to put this in a note box, you can use `.. note::` (search docs for examples on how to do that, is not very common though)